lib/log.dart

```diff
@@ -30,3 +33,45 @@ bool debugLog(String message) {
   }());
   return true;
 }
+
+/// Display an error message.
+///
+/// This shows a [SnackBar] containing the message after app startup,
+/// otherwise logs it to the console.
```

Comment:
Can we be more specific about "after app startup"? I can imagine confused readers wondering how the snackbar could be shown at any time other than after the app starts up.